internal/server/scatter.go

```diff
@@ -0,0 +1,14 @@
+package server
+
+import (
+	"github.com/dicedb/dice/internal/cmd"
+	"github.com/dicedb/dice/internal/comm"
+	"github.com/dicedb/dice/internal/shard"
+)
+
+func scatterPING(mgr *shard.ShardManager, redisCmd *cmd.RedisCmd, c *comm.Client) (cm []cmd.RedisCmd) {
```

Comment:
I don't know if we intend to do this.
Except for MultiSET/GET commands, I imagine a single shard handles all commands. So this would create an array []cm.RedisCmd of size 1.